src/sentry/tasks/sentry_apps.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import, print_function
+
+import six
+from time import time
+
+from sentry.http import safe_urlopen
+from sentry.tasks.base import instrumented_task
+from sentry.utils import json
+from sentry.utils.http import absolute_uri
+from sentry.models import SentryAppInstallation
+from sentry.api.serializers import serialize, app_platform_event
+
+def notify_sentry_app(event, futures):
+    group = event.group
+    project = group.project
+    project_url_base = absolute_uri(u'/{}/{}'.format(
+        project.organization.slug,
+        project.slug,
+    ))
+
+    event_context = serialize(event)
+    event_context['url'] = u'{}/issues/{}/events/{}/'.format(
+        project_url_base,
+        group.id,
+        event.id,
+    )
+    data = {'event': event_context}
+    for f in futures:
+        sentry_app = f.kwargs['sentry_app']
+        try:
+            install = SentryAppInstallation.objects.get(
+                organization=event.project.organization_id,
+                sentry_app=sentry_app,
+            )
+        except SentryAppInstallation.DoesNotExist:
+            return
```

Comment:
If you bail on the first missing app, any futures after the missing one will never be attempted, is that the intended behavior?

Reply:
tis not, will change to use `continue`